src/apps/antioch_kinetic_rates.C

```diff
@@ -98,7 +101,9 @@ int main(int argc, char* argv[])
   while( T < T1 )
     { 
       Antioch::TempCache<libMesh::Real> T_cache(T);
-      
+
+      libMesh::Real rho = p0/(R_mix*T);
+
       antioch_kinetics.omega_dot( T_cache, rho, Y, omega_dot );
```

Comment:
Agreed. I need to dig into what Sylvain has done - I'm not sure we have methods for such calculations in Antioch yet, but in the short term, this is definitely simpler. Will open a new ticket to capture the thought.